test/unit/default-transit-auth-method.bats

```diff
@@ -27,15 +27,15 @@ load _helpers
 #--------------------------------------------------------------------
 # settings
 
-@test "defaultTransitAuthMethod/CR: serviceaccount uses operator sa as a default" {
+@test "defaultTransitAuthMethod/CR: kubernetes default serviceaccount uses operator sa as a default" {
     cd `chart_dir`
     local object=$(helm template \
         -s templates/default-transit-auth-method.yaml  \
         --set 'controller.manager.clientCache.persistenceModel=direct-encrypted' \
         . | tee /dev/stderr)
 
     actual=$(echo "$object" | yq '.spec.kubernetes.serviceAccount' | tee /dev/stderr)
-     [ "${actual}" = "release-name-vault-secrets-operator-controller-manager" ]
+      [ "${actual}" = "release-name-vault-secrets-operator-controller-manager" ]
```

Comment:
nit: would it be possible to shift this over to line up with the line above? I think there are few more below like that as well.